src/transactions/feed/SwapFeedItem.tsx

```diff
@@ -13,64 +14,74 @@ import { Spacing } from 'src/styles/styles'
 import variables from 'src/styles/variables'
 import { useTokenInfo } from 'src/tokens/hooks'
 import TransactionFeedItemImage from 'src/transactions/feed/TransactionFeedItemImage'
-import { TokenExchange } from 'src/transactions/types'
+import { TokenExchange, TokenTransactionTypeV2 } from 'src/transactions/types'
 
 interface Props {
-  exchange: TokenExchange
+  transaction: TokenExchange
 }
 
-function SwapFeedItem({ exchange }: Props) {
+function SwapFeedItem({ transaction }: Props) {
   const { t } = useTranslation()
-  const incomingTokenInfo = useTokenInfo(exchange.inAmount.tokenId)
-  const outgoingTokenInfo = useTokenInfo(exchange.outAmount.tokenId)
+  const incomingTokenInfo = useTokenInfo(transaction.inAmount.tokenId)
+  const outgoingTokenInfo = useTokenInfo(transaction.outAmount.tokenId)
 
-  const handleTransferDetails = () => {
-    navigate(Screens.TransactionDetailsScreen, { transaction: exchange })
+  const handleOpenTransactionDetails = () => {
+    navigate(Screens.TransactionDetailsScreen, { transaction: transaction })
     ValoraAnalytics.track(HomeEvents.transaction_feed_item_select)
   }
 
+  const isCrossChainSwap = transaction.type === TokenTransactionTypeV2.CrossChainSwapTransaction
+
   return (
-    <Touchable testID="SwapFeedItem" onPress={handleTransferDetails}>
+    <Touchable testID="SwapFeedItem" onPress={handleOpenTransactionDetails}>
       <View style={styles.container}>
         <TransactionFeedItemImage
-          status={exchange.status}
-          transactionType={exchange.__typename}
-          networkId={exchange.networkId}
+          status={transaction.status}
+          transactionType={transaction.__typename}
+          networkId={transaction.networkId}
+          hideNetworkIcon={isCrossChainSwap}
```

Comment:
nit: Why aren't we keeping the network icon here? Still seems like a useful bit of info. Or do we have another plan for a future PR?

Reply:
the network icon is hidden [on purpose](https://www.figma.com/design/fjpfHZdVSLazv330kT2sQz?node-id=1573-6584#875927421) from the design side. i also asked this question during a refinement and it was Kayla's preference to hide it for now